README.md

```diff
@@ -124,75 +124,75 @@ To get a local copy up and running, follow these steps.
 
 In order to run this project you need:
 
-<!--
+
 Example command:
 
 ```sh
- gem install rails
+ npm install 
 ```
- -->
+
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
+
 Example commands:
 
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/samuelmoiwa/react-redux-capstone-project
 ```
---->
+-
 
 ### Install
 
 Install this project with:
 
-<!--
+
 Example command:
 
 ```sh
-  cd my-project
-  gem install
+  cd react-redux-capstone-project
+  npm install
 ```
---->
+-
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
+
 Example command:
 
 ```sh
-  rails server
+  npm start
 ```
---->
+-
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
+
 Example command:
 
 ```sh
-  bin/rails test test/models/article_test.rb
+  npm test
 ```
---->
+-
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
+
```

Comment:
- [ ] I think you didn't add the deploy link to your README so kindly deploy your project and add it to your README file ✅🔥

Reply:
- [ ] This issue is not yet resolved :x: 

- As requested by the previous reviewer, please kindly deploy your app and add the live demo link to the readme to make the readme more professional.